packages/backend/test/peripherals/database/ForcedTransactionsRepository.test.ts

```diff
@@ -239,7 +240,9 @@ describe(ForcedTransactionsRepository.name, () => {
     )
 
     const transactions = await repository.getByPositionId(positionId)
-    expect(transactions).toEqual([
+
+    expect(transactions).toBeAnArrayOfLength(2)
```

Comment:
why not use toEqual? you first check if array is of length 2 and later you assert that it has two exact elements. Wouldn't simple toEqual do the same job?

Reply:
We don't sort the elements here, while I was testing it the array had different order and caused the test to fail

Reply:
okay, but you can always sort the just before assertion. I think this would result in a more readable assertion, but it's a nit.

Reply:
Actually, when I think about it rn, we should sort the transactions by lastUpdate, I'll fix it now